test/helpers/cliCommandRunner.js

```diff
@@ -0,0 +1,49 @@
+// Import Node.js Dependencies
+import { fork } from "node:child_process";
+import { createInterface } from "node:readline";
+
+// Import Third-party Dependencies
+import { MockAgent, setGlobalDispatcher } from "undici";
+import stripAnsi from "strip-ansi";
+
+export async function runProcess(options) {
```

Comment:
There is a world where this could make sense to have an AsyncGenerator here to just 

```js
yield stripAnsi(line);
```